apps/ui/src/helpers/turbo.ts

```diff
@@ -0,0 +1,22 @@
+export const MAX_BODY_LENGTH = {
+  default: 10000,
+  turbo: 40000
+};
+
+export const MAX_CHOICES = {
+  default: 500,
+  turbo: 1000
+};
+
+export const MAX_1D_PROPOSALS = {
+  default: 20,
```

Comment:
Maybe we should add unverified space value too

Reply:
what error message should we show in this case? something that says "verify your space"?

Reply:
or do we want to show a turbo message if space hits the unverified limit?

Reply:
We can show the turbo message, so they could jump from unverified to turbo 🤪

Reply:
💵 💵 💵 💵 💵 💵 💵 💵

Reply:
We should show a message: `Get your space verified to increase the limit to ${limit}.`